mlflow/utils/environment.py

```diff
@@ -68,9 +73,9 @@ def _mlflow_additional_pip_env(
         return requirements
 
 
-def _get_additional_pip_dep(conda_env):
+def _get_pip_dep(conda_env):
```

Comment:
Renamed this function since what this function does is just extract pip dependencies in the given conda env.